From 4292a97cb43a80b6a0b960f054f5c3c81a9eb6e2 Mon Sep 17 00:00:00 2001
From: Alicia Klinvex <amklinv@sandia.gov>
Date: Thu, 4 May 2017 12:45:29 -0600
Subject: [PATCH] Fixed build error in C_BaseLib

---
 Src/C_BaseLib/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Src/C_BaseLib/CMakeLists.txt b/Src/C_BaseLib/CMakeLists.txt
index 16802ff80..994850f66 100644
--- a/Src/C_BaseLib/CMakeLists.txt
+++ b/Src/C_BaseLib/CMakeLists.txt
@@ -16,7 +16,7 @@ include_directories(${CBOXLIB_INCLUDE_DIRS})
 set(CXX_source_files Arena.cpp BArena.cpp BaseFab.cpp BLBackTrace.cpp BoxArray.cpp Box.cpp BoxDomain.cpp BoxLib.cpp BoxList.cpp CArena.cpp CoordSys.cpp DistributionMapping.cpp FabArray.cpp FabConv.cpp FArrayBox.cpp FPC.cpp Geometry.cpp MultiFabUtil.cpp IArrayBox.cpp IndexType.cpp IntVect.cpp iMultiFab.cpp MemPool.cpp MultiFab.cpp Orientation.cpp ParallelDescriptor.cpp ParmParse.cpp RealBox.cpp UseCount.cpp Utility.cpp VisMF.cpp)
 set(F77_source_files BLBoxLib_F.f bl_flush.f BLParmParse_F.f BLutil_F.f)
 set(FPP_source_files COORDSYS_${BL_SPACEDIM}D.F SPECIALIZE_${BL_SPACEDIM}D.F)
-set(F90_source_files mempool_f.f90 threadbox.f90 MultiFabUtil_$(BL_SPACEDIM)d.f90)
+set(F90_source_files mempool_f.f90 threadbox.f90 MultiFabUtil_${BL_SPACEDIM}d.f90)
 
 set(CXX_header_files Arena.H Array.H ArrayLim.H BArena.H BaseFab.H BLassert.H BLBackTrace.H BLFort.H BLMap.H BLProfiler.H BoxArray.H BoxDomain.H Box.H BoxLib.H BoxList.H CArena.H ccse-mpi.H CONSTANTS.H CoordSys.H DistributionMapping.H FabArray.H FabConv.H FArrayBox.H FPC.H Geometry.H MultiFabUtil.H IArrayBox.H IndexType.H IntVect.H Looping.H iMultiFab.H MemPool.H MultiFab.H Orientation.H ParallelDescriptor.H ParmParse.H PArray.H PList.H Pointers.H RealBox.H REAL.H SPACE.H Tuple.H UseCount.H Utility.H VisMF.H winstd.H)
 set(F77_header_files)
-- 
2.12.1

